Enable SELinux for Android 14 for IVI

ceiba1985 opened the pull request:
Write essential rules to let Android 14 for IVI boot up with selinux in enforcing mode.

Tracked-On: OAM-117568

ceiba1985 committed:
Update sepolicy rules for graphics

Define specific labels for sysfs and proc files related with
graphics, and grant access permissions to related domains such as
bootanim, surfaceflinger etc.

Tracked-On: OAM-117568
Signed-off-by: jizhenlo <zhenlong.z.ji@intel.com>

ceiba1985 committed:
Update sepolicy rules for audio bt and wlan

Tracked-On: OAM-117568
Signed-off-by: jizhenlo <zhenlong.z.ji@intel.com>

ceiba1985 committed:
Update sepolicy rules for dumpstate and init

Allow dumpstate to read sysfs_health2_0_management to pass CTS.
Allow init to access sysfs_health2_0_management to tune filesystem
IO performance.

Tracked-On: OAM-117568
Signed-off-by: jizhenlo <zhenlong.z.ji@intel.com>

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
SUCCESS: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has started MERGE Build for this pr ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has started Engineering Build for this issue ,Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
FAILURE: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

ceiba1985 committed:
Add debugfs restriction check for debugfs related rules

Some targets enabled debugfs restriction, and the read/write rules
related with debugfs won't pass the neverallow check on these targets.

Tracked-On: OAM-117568
Signed-off-by: jizhenlo <zhenlong.z.ji@intel.com>

sysopenci commented:
Android CI has completed MERGE Build for this pr, build is FAILURE. Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
SUCCESS: Android CI has completed Engineering Build for this issue.Please check the linked Tracked-On issue/Android CI Web  for more details.

sysopenci commented:
Android CI has started MERGE Build for this pr ,Please check the linked Tracked-On issue/Android CI Web  for more details.
